### PR TITLE
build(core): all-around tx check in assertoor

### LINKED ISSUE #1128: Assertoor test must check the tx spammer is working as expected as part of the stability check
Right now the tx spammer is part of the assertoor run but nothing checks it works, after the initial blob tx checks we can check for inclusion of txs in the `stability_check` which doesn't produce txs, if it found them then the tx spammer should be working.

### PR DESCRIPTION
**Pull request by rodrigo-o**
**Motivation**

We want to have more checks regarding transactions, this PR accomplish that through assertoor

**Description**

This PR tackles 2 main things:
- Having a way to validate the tx spammer is working as expected.
- Add [additional transaction checks](https://github.com/lambdaclass/ethrex/blob/f76c491700f1feda7741c7bd6b85d1e34493f49f/test_data/assertoor/el-stability-check.yml#L24-L29) to the assertoor suite to not just validate blob transactions.
  - `eoa-transactions-test` [[Link to playbook](https://github.com/ethpandaops/assertoor/blob/master/playbooks/stable/eoa-transactions-test.yaml)]

To do it, I separated the `network_params.yaml` used for the `make localnet` from 2 new ones used just for the assertoor workflow, one for the blobs + stability check and the other for the new tx checks. 

Also during the stability check we make sure at least one block is proposed by us with more than 50tx validating that the tx spammer is working as expected. The separation was done due to timing, it wasn't possible to parallelize the test as far as I checked without creating a new playbook and I preferred to use the already provided in parallel steps.

Apart from creating the 2 new files, I removed assertoor from the one used by `make localnet`.

**TIME**
Right now the new job takes ~15min, and added to the ~5min for the build it accounts to ~20min.

Closes #1128

### CODE REVIEW

File: test_data/assertoor/network_params_blob.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+additional_services:
+  - assertoor
+  - tx_spammer
+
+assertoor_params:
+  run_stability_check: false
+  run_block_proposal_check: false
+  run_blob_transaction_test: true
+  tests:
+    - 'https://raw.githubusercontent.com/lambdaclass/ethrex/refs/heads/assertoor-allaround-tx-check/test_data/assertoor/el-stability-check.yml'
```
Comment (rodrigo-o) [inline test_data/assertoor/network_params_blob.yaml:21]: After review and before merging `assertoor-allaround-tx-check` will be replaced by `main`.

File: test_data/assertoor/network_params_tx.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+additional_services:
+  - assertoor
+
+assertoor_params:
+  run_stability_check: false
+  run_block_proposal_check: false
+  run_transaction_check: true
+  tests:
+    - https://raw.githubusercontent.com/lambdaclass/ethrex/refs/heads/assertoor-allaround-tx-check/test_data/assertoor/el-big-calldata-check.yml
```
Comment (rodrigo-o) [inline test_data/assertoor/network_params_tx.yaml:20]: Same as https://github.com/lambdaclass/ethrex/pull/1259#discussion_r1857316045

Comment (rodrigo-o) [reply test_data/assertoor/network_params_tx.yaml]: Not in the PR anymore

**Review by fkrause98 — APPROVED** [review]
LGTM!

File: .github/workflows/integration.yaml [hunk 3]
```diff
@@ -55,13 +55,40 @@ jobs:
...
+          docker load --input /tmp/ethrex_image.tar
+
+      - name: Run assertoor
+        uses: ethpandaops/kurtosis-assertoor-github-action@v1
+        with:
+          enclave_name: "ethrex-assertoor-blob"
+          kurtosis_version: "1.4.2"
+          ethereum_package_url: "github.com/lambdaclass/ethereum-package"
+          ethereum_package_branch: "ethrex-integration"
+          ethereum_package_args: "./test_data/assertoor/network_params_blob.yaml"
```
Comment (mpaulucci) [inline .github/workflows/integration.yaml:91]: can we move the ci specific files to be under `.github/something`? `.github/config` maybe? We already have `.github/scripts`

Comment (rodrigo-o) [reply .github/workflows/integration.yaml]: Added them inside `.github/config/assertoor`

File: .github/workflows/ci_skipped.yaml [hunk 4]
```diff
@@ -2,15 +2,11 @@ name: CI Skipped
...
-      - '**/*.sh'
-      - '**/*.json'
-      - '**/*.rlp'
-      - 'Dockefile*'
-
+    paths:
+      - 'README.md'
+      - 'LICENSE'
+      - "**/README.md"
+      - "**/docs/**"
```
Comment (rodrigo-o) [inline .github/workflows/ci_skipped.yaml:9]: This revert a change that spawned CI skipped checks on PRs that didn't need them.

File: test_data/assertoor/network_params_blob.yaml (hunk 1 above)
Comment (rodrigo-o) [reply test_data/assertoor/network_params_blob.yaml]: Done

**Pull request merged**